app/models/service_ansible_playbook.rb

```diff
@@ -45,6 +45,12 @@ def postprocess(action)
     delete_inventory(action) unless use_default_inventory?(hosts)
   end
 
+  def jobs
```

Comment:
@bzwei I had this initially, but I found that my sample data had multiple resources for the same action, which is why I thought I'd utilize job. Is this just an anomaly due to bad data?